### DISCUSSION TITLE
Railgun & Moloch Balancing; or "one-hit-kills and you"

### DISCUSSION THREAD

**Discussion by Ek-Videogames** [FakeFishGames/Barotrauma#7473] [Ideas and suggestions]
With railguns having gotten large buffs across the board in recent updates, it now feels like monsters like moloch, watcher, endworm that have been designed with 'weak point' or combat gimmicks to be _a bit too weak_ compared to the damage output players now have access to.

I think a middleground may be needed where tackling monsters in early-campaign by playing smart and utilizing their weakpoints or other mechanics makes them possible to fight using the 'weaker' coilguns, but isn't completely trivialized by the huge amount of damage a single railgun shell can do when it hits the same weakpoint.

**Comment by Quartofel** [comment]
The moment our group equipped our Dugong of all things with 2x laser and a railgun we've completely stopped worrying about anything but endworm and some of the charm was lost.
